pallets/inflation/Cargo.toml

```
@@ -0,0 +1,50 @@
+[package]
+name = "pallet-inflation"
+version = "4.0.0-dev"
+description = "t3rn pallet for handling token inflation and staking rewards"
+authors = ["t3rn devs <team@t3rn.io"]
+homepage = "https://t3rn.io/"
+license = "Apache 2.0"
+edition = "2018"
+publish = false
+repository = "https://github.com/t3rn/t3rn"
+
+[package.metadata.docs.rs]
+targets = ["x86_64-unknown-linux-gnu"]
+
+[dependencies]
+codec = { package = "parity-scale-codec", version = "2.0.0", default-features = false, features = [
+    "derive",
+] }
+serde = { version = "1.0.136", default-features = false, features = ["derive"] }
```

It might be good not to peg revisions and only to a minor version, so we can get free updates.
```suggestion
serde = { version = "1.0", default-features = false, features = ["derive"] }
```

---

Fixed in upcoming commit